R/result.R

```diff
@@ -23,8 +23,11 @@ result <- function(x, message = NULL, correct = FALSE) {
 }
 
 
-# TODO-document
-# Should be what EVERY check_* returns
+#' \code{grader_graded} object
+#' returned by every \code{check_*} function
```

Comment:
add a blank line above this to separate the title and content.

Reply:
This should state 

"When implementing checking functions to be used by \code{\link{grade_learnr}}, the method should return a \code{graded} value."